terraform/alerting/verification-server/alerts.tf

```diff
@@ -327,3 +327,44 @@ EOT
     google_logging_metric.stackdriver_export_error_count
   ]
 }
+
+# fast error budget burn alert
+resource "google_monitoring_alert_policy" "fast_burn" {
+  project      = var.monitoring-host-project
+  display_name = "Fast error budget burn"
+  combiner     = "OR"
+  enabled      = "true"
+  # create only if using GCLB, which means there's an SLO created
+  count = var.https-forwarding-rule == "" ? 0 : 1
+  conditions {
+    display_name = "2% burn in 1 hour"
+    condition_threshold {
+      filter = "select_slo_burn_rate(\"projects/${var.monitoring-host-project}/services/verification-server/serviceLevelObjectives/availability-slo\", \"3600s\")"
+      duration = "0s"
+      comparison = "COMPARISON_GT"
+      threshold_value = 3.36
```

Comment:
Is this a magic value? Might be a good idea to add a comment explaining this number.

Reply:
Added explanation.